### DISCUSSION TITLE
Is it ok to discuss Kit/Repository dependencies and overarching topics here?

### DISCUSSION THREAD

**Discussion by drcgjung** [eclipse-tractusx/sig-release#201] [Q&A] [answered]
I started some pre-planning discussions to capture dependencies between individual Kits/Use Cases and repositories in eclipse-tractus-x (currently targetted for 24.03)

- https://github.com/eclipse-tractusx/sig-release/discussions/200
- https://github.com/eclipse-tractusx/sig-release/discussions/202
- https://github.com/eclipse-tractusx/sig-release/discussions/203

Is this repo/discussion section the right place for that?

Thx,
CGJ

**Comment by FaGru3n** [comment, marked as answer]
Hi @drcgjung  sure this repo "sig-release" is planned to store all Features for Eclipse Tractus-X as an overarching approch to show what is planned in the upcomming release.

I guess the other discussions look like an Feature description for the issues that should be developed.

In my perspective you can create these as an issue and have to be discussed in the planning rounds... as described on the Repo [ReadMe](https://github.com/eclipse-tractusx/sig-release#issue-process)

**Reply by drcgjung** [reply]
Yes, great. that was the plan ... start with a "session" discussion in "Ideas" between multiple stakeholders during pre-planning and then create issues in the respective teams/topics/repos from that during the planning.
